lib/rules/valid-jsdoc.js

```diff
@@ -279,7 +279,13 @@ module.exports = {
                             hasReturns = true;
 
                             if (!requireReturn && !functionData.returnPresent && (tag.type === null || !isValidReturnType(tag)) && !isAbstract) {
-                                context.report(jsdocNode, "Unexpected @" + tag.title + " tag; function has no return statement.");
+                                context.report({
+                                    node: jsdocNode,
+                                    message: "Unexpected @{{ title }} tag; function has no return statement.",
```

Comment:
I noticed most other messages are doing `{{foo}}` (no whitespaces inside the brackets). I guess it's more of a nitpick, but maybe we do that here too to keep it consistent?